bedevere/prtype.py

```diff
@@ -0,0 +1,46 @@
+"""Label a pull request based on its type."""
+import enum
+import pathlib
+
+from . import util
+
+LABEL_PREFIX = "type"
+
+
+@enum.unique
+class Category(enum.Enum):
+    """Category of Pull Request."""
+    bugfix = f"{LABEL_PREFIX}-bugfix"
+    documentation = f"{LABEL_PREFIX}-documentation"
+    enhancement = f"{LABEL_PREFIX}-enhancement"
+    performance = f"{LABEL_PREFIX}-performance"
+    security = f"{LABEL_PREFIX}-security"
+    tests = f"{LABEL_PREFIX}-tests"
+
+
+async def add_category(gh, issue, category):
+    """Apply this type label if there aren't any type labels on the PR."""
+    if any(label.startswith("type") for label in util.labels(issue)):
+        return
+    await gh.post(issue["labels_url"], data=[category.value])
+
+
+async def classify_by_files(gh, pull_request, filenames):
+    """Categorize pull request."""
+    issue = await util.issue_for_PR(gh, pull_request)
+    docs = tests = 0
+    for filename in filenames:
+        if util.is_news_dir(filename):
+            continue
+        filepath = pathlib.PurePath(filename)
+        if filepath.suffix == '.rst':
+            docs += 1
+        elif filepath.name.startswith('test_'):
+            tests += 1
+        else:
+            return
+    if tests:
+        await add_category(gh, issue, Category.tests)
+        return
```

Comment:
Make this an `if`/`elif` and drop the `return`.

Reply:
Thanks - I had it that way at first and wasn't sure which was the better way to write it.